agent/functional_tests/README.md

```diff
@@ -23,6 +23,11 @@ Before running these tests, you should build the ECS agent and tag its image as
 
 You should also run the registry the tests pull from. This can most easily be done via `make test-registry`.
 
+You should run following commands to set up the IP tables rules:
```

Comment:
Is this needed for all the tests now?

Reply:
Yes I made it required now. 

If I make it optional, we need to have a setting like `TEST_DISABLE_V3_ENDPOINT_BRIDGE_TEST`, and we will result in having too much test environment variables. Since we our test environment has already set up these ip table rules, make it required seems more reasonable.